### PR TITLE
Enable bluetooth from within the app

### PR DESCRIPTION
**Pull request by roalts**
Enables the bluetooth from a dialog to allow or deny to enable bluetooth

### CODE REVIEW

File: src/main/java/com/github/pires/obd/reader/activity/MainActivity.java [hunk 1]
```diff
@@ -531,7 +534,9 @@ protected Dialog onCreateDialog(int id) {
                 build.setMessage(getString(R.string.text_no_bluetooth_id));
                 return build.create();
             case BLUETOOTH_DISABLED:
-                build.setMessage(getString(R.string.text_bluetooth_disabled));
+                Intent enableBtIntent = new Intent(BluetoothAdapter.ACTION_REQUEST_ENABLE);
+                startActivityForResult(enableBtIntent, REQUEST_ENABLE_BT);
+//                build.setMessage(getString(R.string.text_bluetooth_disabled));
```
Comment (pires) [inline src/main/java/com/github/pires/obd/reader/activity/MainActivity.java]: Why leave this commented?

File: src/main/java/com/github/pires/obd/reader/activity/MainActivity.java [hunk 2]
```diff
@@ -659,6 +664,18 @@ public void onGpsStatusChanged(int event) {
         }
     }
 
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+        if (requestCode == REQUEST_ENABLE_BT) {
+            if (resultCode == Activity.RESULT_OK) {
+                btStatusTextView.setText(getString(R.string.status_bluetooth_connected));
+            } else {
+                Toast.makeText(this, "Bluetooth not enabled", Toast.LENGTH_LONG).show();
```
Comment (pires) [inline src/main/java/com/github/pires/obd/reader/activity/MainActivity.java]: Why not reuse `getString(R.string.text_bluetooth_disabled)`?

Comment (roalts) [inline src/main/java/com/github/pires/obd/reader/activity/MainActivity.java]: Yeah that just skipped my mind, I ll make the necessary changes.

**Comment by roalts** [comment]
fixed, sorry for the delay

**Comment by pires** [comment]
Thank you @roalts

**Pull request merged**